src/main/java/build/buildfarm/instance/shard/ShardInstance.java

```diff
@@ -175,6 +175,8 @@
           .register();
   private static final Counter casMissCounter =
       Counter.build().name("cas_miss").help("Number of CAS misses from worker-worker.").register();
+  private static final Counter requeueFailureCounter =
```

Comment:
we increment it every time an operation fails to requeue because of
- blocked
- too many requeues 
- missing

These usually result in a client failure so I'm hoping the metric is rare and indicates bad operations or unhealthy cluster

Reply:
Yeah, sounds good. I just feel this is a very good indicator and we should report it in log and error message whenever possible.